tools/font-edit/README.md

```diff
@@ -14,4 +14,13 @@ make
 ./twin-fedit < nchars
 ```
 
-(press 'q' to next character)
+| Key binding | Function |
+|   ---       |   ---    |
+| `esc`       | Exit program |
```

Comment:
According to [ASCII table](https://theasciicode.com.ar/ascii-control-characters/escape-ascii-code-27.html), the Escape is abbreviated as "ESC" (uppercase) rather than "esc" (lowercase).